rpc/namenode.go

```diff
@@ -145,25 +145,32 @@ func (c *NamenodeConnection) resolveConnection() error {
 	}
 
 	var err error
+	stackErrors := fmt.Errorf("resolveConnection failed. ")
 
 	if c.host != nil {
 		err = c.host.lastError
 	}
 
-	for _, host := range c.hostList {
+	for i, host := range c.hostList {
 		if host.lastErrorAt.After(time.Now().Add(-backoffDuration)) {
+			stackErrors = fmt.Errorf("%sAttempt %v: still backing off host %v after error" +
```

Comment:
Any reason not to use "\n"? we're going to get a very run on string here if we have an issue

Reply:
Good idea.